app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,337 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ */
+
+ @import "bootstrap-sprockets";
+ @import "bootstrap";
+ @import "font-awesome";
+
+ .pagination-wrapper {
+   text-align: center;
+   margin: 0 auto;
+  }
+
+body {
+  padding-top: 90px;
+}
+
+#search-form {
+  text-align: center;
+}
+
+div.input-append {
+  display: table;
+  height: inherit;
+  width: 100%;
+}
+
+#query {
+  display: table-cell;
+  height: 100%;
+  margin: 0;
+  margin-right: 5px;
+}
+
+.input-append .btn {
+  background-color: transparent;
+  padding-top: 8px;
+}
+
+.navbar-brand.nav-logo {
+  font-family: 'Pacifico', cursive;
+  font-size: 25px;
+  color: black;
+}
+
+.navbar {
+  margin-bottom: 0;
+}
+
+#welcome-page {
+  background: image-url("table.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+#feed-page {
+  background: image-url("tablecloth.jpg");
+  background-repeat: repeat;
+}
+
+h1, h2, h3 {
+  text-align: center;
+  font-family: 'Pacifico', cursive;
+}
+
+.filter-box {
+  text-align: center;
+  font-family: 'Pacifico', cursive;
+
+}
+
+.empty-feed {
+  text-align: center;
+  font-family: 'Glegoo', serif;
+}
+
+.results {
+  font-family: 'Glegoo', serif;
+}
+
+.img-circle {
+    border-radius: 50%;
+}
+
+.videoWrapper {
+position: relative;
+padding-bottom: 56.25%; /* 16:9 */
+padding-top: 25px;
+height: 0;
+}
+
+.videoWrapper iframe {
+position: absolute;
+top: 0;
+left: 0;
+width: 100%;
+height: 100%;
+  padding-bottom: 10px;
+}
+
+.tweetWrapper iframe {
+  min-width: 0 !important;
+}
+
+div.transbox {
+  margin-top: 15%;
+  background-color: #fbf7fb;
+  border: 1px solid black;
+  opacity: 0.95;
+  filter: alpha(opacity=95); /* For IE8 and earlier */
+}
+
+p.welcome {
+  text-align: center;
+  padding: 5px;
+  font-family: 'Pacifico', cursive;
+  font-size: 44px;
+}
+
+p.about-fancy-feed {
+  text-align: center;
+  padding: 10px;
+  font-family: 'Glegoo', serif;
+}
+
+p.login-button {
+  font-family: 'Glegoo', serif;
+}
+
+.login-links {
+  padding-top: 5px;
+  text-align: center;
+}
+
+.login-links .btn {
+  background-color: transparent;
+  border: none;
+}
+
+.login-links p {
+  margin-bottom: 0;
+  text-align: bottom;
+}
+
+.sticky-footer {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  background: rgb(248, 248, 248);
```

Comment:
Is there a reason you're using hex colors in some places and rgb in others? It seems inconsistent.